tests/test_config/test_config.py

```diff
@@ -556,6 +562,23 @@ def test_multi_line_copyright(source_date_year, app, monkeypatch):
         ) in content
 
 
+@pytest.mark.parametrize('copyright_line', [
+    # '1970',
```

Comment:
I think we should still consider this one. It might happen that people write things like that. So it would be good to handle possibilities like that. 

It doesn't matter whether it's legally right or not. What matters is that we still handle inputs that may not always follow strict protocols. Otherwise, we need to describe (in the docs) the exact copyright format we expect.